docs/configuration/inter_node_data_path.md

```diff
@@ -137,3 +137,33 @@ paths = arch.search_paths(
     node_filter = node_filter,
     communication_filter = comm_filter)
 ```
+
+### Combine paths
+
+If the path from `source node` to `destination node` is too long, `Architecture.search_paths()` takes a long time and may find many paths that don't need to be considered.
+It may be laborious to select a correct target path.
```

Comment:
If you remove L. 143, this should be removed.

```suggestion
```

Reply:
I modified as suggestion.